src/mdb_v8.c

```diff
@@ -706,6 +721,11 @@ autoconfigure(v8_cfg_t *cfgp)
 		failed++;
 	}
 
+	if (V8_TYPE_JSTYPEDARRAY == -1) {
```

Comment:
No, it's OK for older versions of node not to have `JSTypedArray`. Will fix too.